package.json

```diff
@@ -31,14 +31,14 @@
     "format:md": "prettier --write '**/*.md'",
     "format:ts": "prettier --write '**/*.ts{,x}'",
     "lint": "eslint src test --ext .ts --fix && yarn format",
-    "test": "yarn build && yarn lint && jest && yarn clean",
```

Comment:
Running this causes the test suite to run twice, once in test and once in the build. 

Also I personally expect the base test script in a package.json to only run the test suite, instead of doing multiple things.